charts/application/values.yaml

```
@@ -56,6 +56,9 @@ readinessProbe:
 image:
   repository: quay.io/heubeck/examiner
   tag: 1.12.21
+  # When hash is set image.tag, imagepolicy and imagerepository is ignored
+  # e.g. "sha256:b187aed68cc8c75f475efe8aebcc61fc7ca0c5adc2d2c5b8318997c3edc8d32f"
+  hash:
```

```suggestion
  digest:
```
see also https://docs.docker.com/reference/cli/docker/image/ls/#digests